CoEpi/ui/onboarding/AttributedString.swift

```diff
@@ -0,0 +1,71 @@
+import UIKit
+
+typealias FontAttributes = [NSAttributedString.Key: Any]
+
+struct AttributableString {
+    
+    let baseFont: UIFont
+    var storage = NSMutableAttributedString()
+    
+    let boldAttributes: FontAttributes
+    let normalAttributes: FontAttributes
+
+    init(baseFont: UIFont) {
+        
+        self.baseFont = baseFont
+        
+        let pointSize = baseFont.pointSize
+        let fontDescriptor = baseFont.fontDescriptor.withSymbolicTraits(.traitBold)
+            ?? UIFont.boldSystemFont(ofSize: pointSize).fontDescriptor
+        
+        let boldFont = UIFont(descriptor: fontDescriptor, size: pointSize)
+        
+        self.boldAttributes = [
+            .font: boldFont
+        ]
+        
+        self.normalAttributes = [
+            .font: baseFont
+        ]
+        
+    }
+    
+    func append(_ value: String, attributes: FontAttributes) {
+        let attributedString = NSAttributedString(string: "\(value) ", attributes: attributes)
+        storage.append(attributedString)
+    }
+    
+    func bold(_ value: String) -> AttributableString {
+        
+        append(value, attributes: boldAttributes)
+        
+        return self
+        
+    }
+    
+    func normal(_ value: String) -> AttributableString {
+        
+        append(value, attributes: normalAttributes)
+        
+        return self
+        
+    }
+    
+    func finish() -> NSAttributedString {
+        
+        let range = NSRange(location: 0, length: storage.length - 1)
+        return storage.attributedSubstring(from: range)
+        
+    }
+    
+}
+
+extension AttributableString {
+    
+    static func avenir(size: CGFloat) -> UIFont {
+        
+        return UIFont(name: "AvenirNext-Regular", size: size)!
```

Comment:
No need for `return` keyword here

Reply:
As with the above I'm unhappy with this class as a whole. Fixing fonts with magic strings in forced unwrapped optionals. Hmm.